medcat/cat.py

```diff
@@ -871,7 +901,7 @@ def multiprocessing(self,
                 If set once more than out_split_size_chars are annotated
                 they will be saved to a file (save_dir_path) and the memory cleared. Recommended
                 value is 20*batch_size_chars.
-            save_dir_path(`str`, None):
+            save_dir_path(`str`, defaults to the directory of the script being run):
```

Comment:
will this go to where the script is located or CWD? should go to CWD

Reply:
I thought the intended "./" meant the script directory. Done.